davi.py

```diff
@@ -0,0 +1,7 @@
+import pandas as pd
+
+def feature_extract(df:pd.DataFrame) -> pd.DataFrame:
+      """Extracts 'Fireplaces', 'HalfBath', 'BedroomAbvGr', 'TotRmsAbvGrd' where 
+      KitchenAbvGr is excluded due to its negative correlation to sale price being inversely related to proper sales price predictions"""
```

Comment:
I'm of the opinion that even a negative correlation is a correlation. If 'rats' was a feature, it would probably be negatively correlated to sale price, right? But that would still be important information.

I'm for adding it back in, personally, though I won't withhold approval for it.

Reply:
Added it back in. We may want to handle it with some binning in the future